src/foraging_gui/Foraging.py

```diff
@@ -3368,11 +3406,14 @@ def log_git_hash():
         Add a note to the GUI log about the current branch and hash. Assumes the local repo is clean
     '''
     try:
-        git_hash = subprocess.check_output(['git','rev-parse','--short', 'HEAD']).decode('ascii').strip()
```

Comment:
Using `git rev-parse --short HEAD` takes 50ms on my computer compared to 870ms for `git log -n 1`

Reply:
"git rev-parse --short HEAD" only gets the head 7 digits.

Reply:
I updated the command to `git rev-parse HEAD`.